vmm/src/device_manager.rs

```diff
@@ -492,6 +492,7 @@ pub fn create_pty(non_blocking: bool) -> io::Result<(File, File, PathBuf)> {
             .open("/dev/ptmx")?,
     };
     let mut unlock: libc::c_ulong = 0;
+    // SAFETY: FFI call into libc, trivially safe
```

Comment:
Just to clarify, you don't mean that all FFI calls into libc are safe, and the call to libc::ioctl here is trivially safe as the input fd is valid and the return value is checked. Right?

Reply:
Yes. That's right.